docs/user-guide/README.md

```diff
@@ -50,6 +50,7 @@ If you don't have much familiarity with Kubernetes, we recommend you read the fo
 1. [Deploying continuously running applications](deploying-applications.md)
 1. [Connecting applications: exposing applications to clients and users](connecting-applications.md)
 1. [Working with containers in production](production-pods.md)
+1. [Migrating from docker-cli to kubectl](docker-cli-to-kubectl.md)
```

Comment:
Actually, not here. This set of docs was written with an intended flow. I suggested under Further reading, below. We could also put a link in quick-start.md.